src/main/java/org/tctalent/anonymization/mapper/DocumentMapper.java

```diff
@@ -60,4 +62,8 @@ default Integer extractYearFromLocalDate(LocalDate dob) {
     return dob != null ? dob.getYear() : null;
   }
 
+  @Named("mapEnumToString")
+  default String mapEnumToString(SurveyType surveyType) {
```

Comment:
This isn't mapping from an enum (SurveyType is not an enum). Can we rename the method to mapSurveyTypeToString?

```suggestion
  default String mapSurveyTypeToString(SurveyType surveyType) {
```